tests/secrets.sample.yaml

```diff
@@ -19,6 +19,8 @@ cinder_password: "{{ lookup('file', tripleo_passwords) | from_yaml | community.g
 glance_password: "{{ lookup('file', tripleo_passwords) | from_yaml | community.general.json_query('*.GlancePassword') | first }}"
 ironic_password: "{{ lookup('file', tripleo_passwords) | from_yaml | community.general.json_query('*.IronicPassword') | first }}"
 neutron_password: "{{ lookup('file', tripleo_passwords) | from_yaml | community.general.json_query('*.NeutronPassword') | first }}"
+heat_password: "{{ lookup('file', tripleo_passwords) | from_yaml | community.general.json_query('*.HeatPassword') | first }}"
+heat_auth_encryption_key: "{{ lookup('file', tripleo_passwords) | from_yaml | community.general.json_query('*.HeatAuthEncryptionKey') | first }}"
```

Comment:
This may need wiring into CI to get it passing. Traditionally we've set each password separately there, but now that we have the default lookup-based method, we could utilize the `tripleo_passwords` file in CI too perhaps, and then we wouldn't have to wire in each new service separately. WDYT @marios @cescgina ?

Those can be separate concerns though. We can first wire in the heat passwords the old way and then switch everything to `tripleo_passwords` file if applicable.

Reply:
sounds good to me, I crated https://review.rdoproject.org/r/c/rdo-jobs/+/50563 to unblock the merge and we can simplify the passwords handling in the future

Reply:
Testing here: https://review.rdoproject.org/r/c/testproject/+/50564

Reply:
:+1: thanks